exchanges/kucoin/kucoin_test.go

```diff
@@ -1999,6 +2001,20 @@ func verifySubs(tb testing.TB, subs subscription.List, a asset.Item, prefix stri
 	}
 }
 
+var subPairs currency.Pairs
```

Comment:
verify subs above seems unused and there are test failures in this file.

Reply:
Thanks

Fixed 54f927f